### PR TITLE
WIP moving content into repo

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**What**
Moving content from seperate repo into one repo

- [x] move content to /content
- [x] move site to /docs folder
- [x] add .gitignore file to ignore /content/public-site (generated for testing)
- [x] update github pages setting to pull from master /docs folder
- [x] Write README
- [x] Script to update 

@JameelB

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
The last commit `building changes` contains site changes relating to [this pr](https://github.com/feedhenry/student-help-guide-content/pull/23)

**Comment by laurafitzgerald** [comment]
@JameelB There's one item left to do for this pr which is to update the repo to source the site from the `docs` folder. Would you mind doing a verification first. 
**Verification Steps**
1.
Run `build-local.sh`
Verify this builds the site locally to content/public site
Verify that the buddy and general rules are now in book/md format rather than pdf
2.
Run `build.sh`
Verify that there are no changes to the contents of the docs folder as i've already built and pushed the required changes.
Verify that there are no errors
Verify that the site is built correctly as above using the built site in the docs/ folder.

Once these are verified and the pr merged I will update the setting to have github pages source the docs/ folder.

File: build-local.sh
Comment (JameelB) [inline build-local.sh]: Can we have this as a pre-requisite in the `README` file rather than having to install tutors each time we try to build it?

File: README.md
Comment (JameelB) [inline README.md]: `contains any assets (images, documents, etc.) and contents written in markdown` wdyt?

File: build.sh
Comment (JameelB) [inline build.sh:11]: I don't think there's a need for the `public-site/` folder if we're copying it to the docs. We can probably remove this after it's been copied so we don't have any duplicates. With the comment above, we won't need the `build-local.sh` file. We can move the `tutors` command here to generate the site.

File: README.md
Comment (JameelB) [inline README.md]: typo on `changes`. 

Can probably rephrase this into something like:
`Once the site has been built, it can be published. In order to publish the site, create a PR with your changes and request a review. Once approved and merged, the website will update automatically.` wdyt?

**Review by JameelB — COMMENTED** [review]
Just a few comments. Tested the scripts and verified that the site is generated correctly. Site is rendering correctly with all the expected contents.

File: build-local.sh
Comment (laurafitzgerald) [reply build-local.sh]: yes definitely

**Comment by laurafitzgerald** [comment]
@JameelB made changes as per your comments if you have time for another review.

File: .gitignore
Comment (JameelB) [inline .gitignore]: I think we can remove this folder after copying to avoid duplicates.

**Review by JameelB — APPROVED** [review]
Just one small comment there but everything looks good to me now. Verified the following changes:
- [x] Buddy Guide and General Rules are now in book format.
- [x] Site is built successfully by the `build.sh` script.
- [x] Site is rendered as expected.

File: .gitignore
Comment (laurafitzgerald) [reply .gitignore]: yes agreed. I was thinking it would be handy to have for testing local changes (it's gitignored) but as the build goes into the docs folder that should be sufficient. Will add that now.

**Comment by JameelB** [comment]
@laurafitzgerald The approval is still valid so feel free to merge

**Comment by laurafitzgerald** [comment]
@JameelB merged. will now change the github pages setting to read from /docs folder

**Pull request merged**
